models/progresses.js

```
@@ -146,11 +146,95 @@ const addUserDetailsToProgressDocs = async (progressDocs) => {
+        ...check.progressData, 
+        createdAt: createdAtTimestamp, 
+        date: progressDateTimestamp 
+      };
+      
+      const docRef = progressesCollection.doc();
+      batch.set(docRef, progressDocumentData);
+      
+      result.successCount++;
+      result.successfulRecords.push({
+        id: docRef.id,
+        ...progressDocumentData
```

ESLint

[error] Insert `,`

(prettier/prettier)